src/grabEbookFromPacktTask.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+function log {
+	echo  $(date +"%Y%m%d_%H%M%S") : $@
+}
+
+typeset -i sleepTime=120
+
+typeset -i i=0
+until [  $i -gt 10 ]; do
+ log grab book 
+ cd /home/pi/Packt/src && /usr/bin/python3 packtPublishingFreeEbook.py -g
```

Comment:
You use hardcoded value here (that's bad), you also assume that we use system Python for that (and that seems like a bad practice).